<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm</class>
 <widget class="QMainWindow" name="Frm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Software Lion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CONSULTA RUC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRazonSocial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscarRUC">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Raz贸n Social:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBuscarRuc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>155</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre Comercial</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombreComercial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>208</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstado">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>228</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>208</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Condici贸n</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCondicion">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>228</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDireccion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Direcci贸n</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>316</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Departamento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDepartamento">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>336</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtProvincia">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>336</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>316</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Provincia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>316</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distrito</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDistrito">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>336</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTipoContribuyente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>361</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de Contribuyente</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFechaInscripcion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>441</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>421</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha Inscripci贸n</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>421</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de Inicio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFechaInicio">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>441</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>474</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Actividad Principal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtActividadPrincipal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>494</y>
      <width>361</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtActividadSecundaria">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Actividades secundarias</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
